Backend/src/main/java/com/example/Backend/services/LoginService.java

```diff
@@ -0,0 +1,44 @@
+package com.example.Backend.services;
+
+import com.example.Backend.entity.UserRegisterDetails;
+import com.example.Backend.model.UserLogin;
+import com.example.Backend.respository.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class LoginService {
+    UserRepository userRepository;
+    LoginService(UserRepository userRepository)
+    {
+        this.userRepository = userRepository;
+    }
+    public UserLogin getUserDetails(String name)
+    {
```

Comment:
Do you think we should move the exception handling to service class? Yes/No. and why?

Reply:
i think we should move the exception handling to service class as controller responsibility is for accepting request. and service class has business logic so if we use service directly in any other part of project then it will handle exception in such scenario.

Reply:
I do not see any exception handling here as you said.